docs/rules/prefer-ternary.md

```diff
@@ -120,3 +120,26 @@ if (test) {
 	baz = 2;
 }
 ```
+
+## Options
+
+Type: `string`\
+Default: `always`
+
+- `always` (default)
+	- Always check `IfStatement`
+- `only-single-line`
+	- Only check if `test`, `consequent`, and `alternate` of `IfStatement` are single line
```

Comment:
```suggestion
- `always` (default)
	- Always report when using an `IfStatement` where a ternary expression can be used.
- `only-single-line`
	- Only check if the content of the `IfStatement` is less than one line long.
```
Maybe we loose a bit in precision (I did not tell about `test`, `consequent` and `alternate`) but I think we gain in clarity as it uses broader terms.
I'm not bilingual though so there still might be a better way to formulate it 😄

Reply:
The content seems not clear, maybe   ``the content of `if` and `else` ``?

Reply:
You're right, do you want me to edit my suggestion or do will you do a commit ?

Reply:
You  can  edit, or  maybe

`` the content of the condition,  `if` and `else` block ``.?

Reply:
what about ``Only check if the content of the `if` and/or `else` block is less than one line long.``?